Library/Formula/aws-shell.rb

```diff
@@ -0,0 +1,169 @@
+class AwsShell < Formula
+  desc "Integrated shell for working with the AWS CLI."
+  homepage "https://github.com/awslabs/aws-shell"
+  url "https://pypi.python.org/packages/source/a/aws-shell/aws-shell-0.1.0.tar.gz"
+  sha256 "2d89e3c51e6cf7da74c8bca8b8a986cc8edcad27ccd6c15678cd84b970f63a16"
+
+  head do
+    url "https://github.com/awslabs/aws-shell.git"
+
+    resource "awscli" do
+      url "https://github.com/aws/aws-cli.git", :branch => "develop"
+    end
+
+    resource "boto3" do
+      url "https://github.com/boto/boto3.git", :branch => "develop"
+    end
+
+    resource "botocore" do
+      url "https://github.com/boto/botocore.git", :branch => "develop"
+    end
+
+    resource "colorama" do
+      url "https://github.com/tartley/colorama.git", :tag => "v0.3.3", :revision => "5906b2604223f3a3bdf4497244fc8861b89dbda6"
```

Comment:
It's less important on the `branch` ones, but the longer ones with multiple conditionals should be split:

```ruby
resource "colorama" do
  url "https://github.com/tartley/colorama.git",
      :tag => "v0.3.3",
      :revision => "5906b2604223f3a3bdf4497244fc8861b89dbda6"
end
```
And such.